client/src/templates/Challenges/video/Show.js

```diff
@@ -233,6 +233,7 @@ export class Project extends Component {
                       // index should be fine as a key:
                       <label className='video-quiz-option-label' key={index}>
                         <input
+                          aria-label='Answer'
```

Comment:
Was this line meant to be added on this PR?

Reply:
No. It wasn't, I am in the middle of a rebase here. And needed to open up the PR to trigger the actions which were showing failed status for some reason. 

Will label the PR QA ready when I am able.

Thanks for your patience.

Reply:
I now see that it was picked up due to you capturing @ojeytonwilliams commits from somewhere else?

Reply:
The commits were added by Oliver on purpose.

Reply:
The new Gatsby version uses more eslint rules, so I added them to ours and made some fixes.